app/services/send_admin_email.rb

```diff
@@ -5,14 +5,30 @@ class SendAdminEmail
   include Service
 
   def call(user:)
+    # returns an array [false, true] or [true, false]
     previous_admin_status = user.previous_changes['admin']
-    send_new_admin_email(user: user) if previous_admin_status[1]
+    send_new_admin_email(user: user)
+
+    if previous_admin_status[1]
+      send_new_admin_email(user: user)
+    elsif previous_admin_status[1] == false
```

Comment:
Let's change that to `!previous_admin_status[1]` - I'm surprised rubocop didn't yell about that

Reply:
That's cleaner. Thanks.